client/my-sites/subscribers/components/subscribers-header-popover/subscribers-header-popover.tsx

```diff
@@ -0,0 +1,63 @@
+import { Gridicon } from '@automattic/components';
+import classNames from 'classnames';
+import { translate } from 'i18n-calypso';
+import { useCallback, useRef, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import PopoverMenu from 'calypso/components/popover-menu';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
+import { addQueryArgs } from 'calypso/lib/url';
+import { recordGoogleEvent } from 'calypso/state/analytics/actions';
+
+import '../shared/popover-styles.scss';
+
+type SubscribersHeaderPopoverProps = {
+	siteId: number | null;
+};
+
+const SubscribersHeaderPopover = ( { siteId }: SubscribersHeaderPopoverProps ) => {
+	const [ isVisible, setIsVisible ] = useState( false );
+	const dispatch = useDispatch();
+	const onToggle = useCallback( () => setIsVisible( ( visible ) => ! visible ), [] );
+	const buttonRef = useRef< HTMLButtonElement >( null );
+	const downloadCsvLink = addQueryArgs(
+		{ page: 'subscribers', blog: siteId, blog_subscribers: 'csv', type: 'email' },
+		'https://dashboard.wordpress.com/wp-admin/index.php'
+	);
+	const onDownloadCsvClick = () => {
+		dispatch(
+			recordGoogleEvent(
+				'Subscribers',
+				'Clicked Download email subscribers as CSV menu item on Subscribers'
+			)
+		);
+	};
+
+	return (
+		<div className="subscriber-popover__container">
+			<button
+				className={ classNames( 'subscriber-popover__toggle', {
+					'is-popover-visible': isVisible,
+				} ) }
+				onClick={ onToggle }
+				ref={ buttonRef }
+			>
+				<Gridicon icon="ellipsis" size={ 24 } />
+			</button>
+
+			<PopoverMenu
+				position="bottom left"
+				onClose={ () => setIsVisible( false ) }
+				isVisible={ isVisible }
+				context={ buttonRef.current }
+				className="subscriber-popover"
+				focusOnShow={ false }
+			>
+				<PopoverMenuItem href={ downloadCsvLink } onClick={ onDownloadCsvClick }>
+					{ translate( 'Download email subscribers as CSV' ) }
```

Comment:
The design on Figma shows `Download CSV` and the implementation has `Download email subscribers as CSV`. Is there an agreement about this button label (considering the note about the CSV only with email subscribers)?

Reply:
You are right. But, honestly, "Download CSV" makes the impression that all the users in the list will be downloaded when the only ones being downloaded are the email subscribers. I think `Download email subscribers as CSV` is more accurate on what the action will do.

Reply:
For now we leave the copy like this (discussed here: p1687337763769789-slack-C02TCEHP3HA )